mantis-server/mantis-server-agent/build.gradle

```diff
@@ -15,8 +15,21 @@
  */
 
 dependencies {
-    compile project(":mantis-control-plane:mantis-control-plane-core")
+    implementation project(":mantis-runtime-loader")
+    implementation project(":mantis-server:mantis-server-worker-client")
+
+    compile libraries.vavr
+
+    implementation libraries.zip4j
+    implementation libraries.commonsIo
+    implementation libraries.hadoopCommon
+    implementation libraries.spotifyFutures
 
     implementation libraries.slf4jApi
     implementation libraries.slf4jLog4j12
+
+    testImplementation libraries.junit4
+    testImplementation libraries.mockitoAll
+    testImplementation project(":mantis-server:mantis-server-worker")
```

Comment:
Is there a reason why we depend on mantis-server-worker here? Shouldn't it be the other way around? In that, I expect fixtures from this module to be used from within the mantis-server worker.

Reply:
"TaskExecutor" is in server-agent and there is a integration test for it and the test requires full worker as testImplementation to run.